src/main/scala/cromwell/binding/AstTools.scala

```diff
@@ -157,7 +157,10 @@ object AstTools {
     would have its own MemberAccess - "a.b.c" and "a.b.d"
   */
   def findTopLevelMemberAccesses(expr: AstNode): Iterable[Ast] = expr.findAstsWithTrail("MemberAccess").filterNot {
-    case(k, v) => v.exists{case a:Ast => a.getName == "MemberAccess"}
+    case(k, v) => v exists {
```

Comment:
two case statements on one line?  how do you propose I do that?  Did you have a different way to write it in mind?